scss/_patterns_navigation.scss

```diff
@@ -40,6 +40,10 @@
   position: relative;
   width: 100%;
 
+  @media (min-width: $breakpoint-medium) {
+    border-bottom: 1px solid $border-color;
```

Comment:
The spec says that only light navigation should have a border on small screens? Does this need updating? https://github.com/ubuntudesign/vanilla-design/blob/master/Navigation/navigation.md